### ISSUE TITLE
Crash after close 

### ISSUE THREAD

**Issue by Dmi3ryd** [MrKepzie/Natron#1776]
Hello.

### Problem
After closing the Natron a very strange error occurs, the program crashes.
This happens every time I open any project. How to fix it?

Describe the problem or suggestion here.

Natron Version 2.3.14
Copyright (C) 2013-2018 INRIA and Alexandre Gauthier-Foichat
Use the --help or -h option to print usage.<<<
Info: init.py script not loaded (this is not an error)
Info: initGui.py script not loaded (this is not an error)
Loading project: D:/Compose_01.ntp

 
### Steps to Reproduce
Just open nes scene, import few images and try to close. Crash!

### Versions
Natron 2.3.14
Win10 64x, 32 Ram

![image](https://user-images.githubusercontent.com/25366819/58004051-eab24b80-7aea-11e9-8c9c-87330e5bd400.png)

**Comment by devernay** [comment]
This is a known issue on windows. Once a windows developer jumps it, we may be able to better understand what happens.
It may be related to https://github.com/OpenImageIO/oiio/issues/1795 in which case it would be solved by upgrading OIIO.
Does it also crash if you build a graph without a reader or writer?

**Comment by Dmi3ryd** [comment]
This does not happen if the project is empty.

**Comment by devernay** [comment]
What about a non-empty  project that has neither readers nor writers: use generators instead of readers.

**Comment by Dmi3ryd** [comment]
**Generators**? It's node?

**Comment by devernay** [comment]
generators are nodes that produce an output without an input, eg checkerboard, radial, constant, colorwheel, etc.

**Comment by Dmi3ryd** [comment]
Not sure if this can be a good solution, because I need to work with real image files.

**Comment by devernay** [comment]
I'm not proposing a workaround, I'm trying to figure out the source of the bug

**Comment by Dmi3ryd** [comment]
I think I wrote it above, right? )

**Comment by devernay** [comment]
Let me rephrase it: do you still get the same behavior (crash after quit) if you edit a project with only generators (see above) and no writer (only one or several viewers).

**Comment by Dmi3ryd** [comment]
With only generators and without reades no errors!

**Comment by devernay** [comment]
Please re-file this issue on the right github, see https://github.com/MrKepzie/Natron/issues/1766
